### PR TITLE
Fix: Nested Tabs - Avoid jump in tab width [ed-8914]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 1]
```diff
@@ -1089,39 +1088,52 @@ protected function content_template() {
 			var elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
 			<div class="e-n-tabs-heading" role="tablist">
 				<# _.each( settings['tabs'], function( item, index ) {
-				var tabCount = index + 1,
-				tabUid = elementUid + tabCount,
-				tabTitleKey = 'tab-title-' + tabUid;
-				tabIcon = elementor.helpers.renderIcon( view, item.tab_icon, { 'aria-hidden': true }, 'i' , 'object' );
+					let tabCount = index + 1,
```
Comment (hein-obox) [inline modules/nested-tabs/widgets/nested-tabs.php:1091]: I had to change the icon display to the change the icon display behaviour.

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 2]
```diff
@@ -1058,8 +1058,7 @@ protected function render() {
 			}
 
 			$tabs_title_html .= "<div {$title_render_attributes}>";
-			$tabs_title_html .= "\t<span {$tab_icon_attributes}> {$icon_html}</span>";
-			$tabs_title_html .= "\t<span {$tab_icon_active_attributes}> {$icon_active_html}</span>";
+			$tabs_title_html .= "\t<span {$tab_icon_attributes}>{$icon_html}{$icon_active_html}</span>";
```
Comment (hein-obox) [inline modules/nested-tabs/widgets/nested-tabs.php:1061]: I had to change the icon display to the change the icon display behaviour.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 3]
```diff
@@ -150,16 +158,16 @@ test.describe( 'Nested Tabs tests', () => {
 		await editor.publishAndViewPage();
 
 		// Set published page variables
-		const icon = await page.locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon svg' ).first(),
-			activeTabIcon = await page.locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon.e-active svg' ).first(),
+		const icon = await page.locator( '.elementor-widget-n-tabs .e-n-tab-title .e-n-tab-icon svg:first-child' ).first(),
```
Comment (hein-obox) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:161]: I had to update the test, after changing the active tab behaviour.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 4]
```diff
@@ -31,13 +31,16 @@
 
 	--n-tabs-title-color: var( --e-global-color-secondary, var( --n-tabs-color-secondary-fallback ) );
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:32]: Comment: I had to branch of this task from https://elementor.atlassian.net/browse/ED-8915, to be able to check the full title alignment behaviour.

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 1 above)
Comment (davseve) [reply modules/nested-tabs/widgets/nested-tabs.php:1091]: Is the display behaviour is changed only in the editor?

Comment (hein-obox) [reply modules/nested-tabs/widgets/nested-tabs.php:1091]: @davseve I have changed it in the php as well, see: https://github.com/elementor/elementor/blob/efa4c5a86fbe567424de046f947222ed5645da51/modules/nested-tabs/widgets/nested-tabs.php#L1061.

**Review by davseve — DISMISSED** [review]
Looks great, 
Consider solving the conflicts on your side before moving forward to Code review.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 5]
```diff
@@ -119,60 +129,61 @@
 			.e-n-tab-icon {
 				display: flex;
 				align-items: center;
+				flex-direction: column;
 				order: var( --n-tabs-icon-order );
+				overflow: hidden;
```
Comment (davseve) [inline modules/nested-tabs/assets/scss/frontend.scss:134]: You are cutting the icon it's bigger than its container?

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 6]
```diff
@@ -106,8 +106,18 @@
 			width: var( --n-tabs-title-width );
 
 			&,
-			span * {
-				transition: all var( --n-tabs-title-transition );
+			span svg,
+			span i {
```
Comment (davseve) [inline modules/nested-tabs/assets/scss/frontend.scss:110]: This is way better the asterisk.
Why is this block comma separated?

**Review by davseve — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/assets/scss/frontend.scss (hunk 5 above)
Comment (hein-obox) [reply modules/nested-tabs/assets/scss/frontend.scss:134]: I am hidden the active/non-active icon.

The icon itself will alway take on the size of the container.

**Review by TzviRabinovitch — APPROVED** [review]

**Review by davseve — APPROVED** [review]

**Pull request merged**